setup.py

```diff
@@ -17,26 +17,26 @@
               author_email="",
               url="https://github.com/CNES/rastertools",
               packages=find_packages(exclude=['tests']),
+              package_data={"eolab.georastertools": ["**/*.json"]},
               include_package_data=True,
               zip_safe=False,
-              setup_requires = ["setuptools_scm"],
+              setup_requires=["setuptools_scm"],
               install_requires=[
                   'click==8.1.8',
                   'rasterio',
                   'pytest>=3.6',
                   'pytest-cov',
-                  'geopandas==0.13',
+                  'geopandas>=0.13',
                   'kiwisolver==1.4.5',
                   'matplotlib==3.7.3',

```

Comment:
There are other dependencies pinned down, is this necessary?

Reply:
Yes I would prefer to have just minimum requirements if possible, I don't like hard pinning like that, so if you can replace all `==` by `>=` and check that CI is okay that would be perfect!